src/core/symbology-ng/qgsfillsymbollayerv2.h

```diff
@@ -550,7 +550,7 @@ class CORE_EXPORT QgsImageFillSymbolLayer: public QgsFillSymbolLayerV2
     virtual ~QgsImageFillSymbolLayer();
     void renderPolygon( const QPolygonF& points, QList<QPolygonF>* rings, QgsSymbolV2RenderContext& context ) override;
 
-    virtual QgsSymbolV2* subSymbol() override { return mOutline; }
+    virtual QgsSymbolV2* subSymbol() const override { return mOutline; }
```

Comment:
shouldn't these be "const QgsSymbolV2* " ? Otherwise the method is lying when it's reporting that it's const.